src-frontend/app/controllers/procedure.js

```diff
@@ -44,17 +57,20 @@ export default Ember.Controller.extend({
         updateSortOrder: function(pageModels) {
             var controller = this;
 
+            pageModels = pageModels || controller.generateNewPageIndices()
```

Comment:
it's local, passed into the function on line 57

Do we even have JS linter checks on builds?